client/src/app/shared/directives/os-perms.directive.ts

```diff
@@ -1,33 +1,33 @@
 import { Directive, Input, ElementRef, TemplateRef, ViewContainerRef, OnInit } from '@angular/core';
```

Comment:
ElementRef and OnInit
are not used anymore. Please remove while you are on it ;-)